.gitignore

```diff
@@ -142,7 +142,7 @@ pnpm-lock.yaml
 yarn.lock
 
 # editor files
-# .vscode
+.vscode
```

Comment:
I would keep ignoring the directory. One of the reasons I wrote the guide the way I did is so that we wouldn't have to include this directory at all. The editor is going to update any file in the directory, and will result in extraneous data in PRs.

Reply:
The `.vscode/extensions.json` is the one useful file

Reply:
Oh, I misread it. Yeah, that suggestion makes sense. 👍